src/components/MoviesApp/FetchMovies/FetchMovies.jsx

```diff
@@ -0,0 +1,5 @@
+
+export default function FetchMovies(value, option) {
```

Comment:
It's not a component but regular function, so it must be in helpers file and has name in lower case.

```
MovieApp/helpers.js

export default function fetchMovies(value, option) {
```